src/ruby/ext/grpc/rb_grpc.h

```diff
@@ -78,4 +80,10 @@ void grpc_rb_fork_unsafe_end();
 
 void grpc_ruby_init();
 
+#define GRPC_RUBY_ASSERT(x)                 \
+  if (!(x)) {                               \
+    fprintf(stderr, "%s", "Assert failed"); \
```

Comment:
It would be useful to keep file and line number in particular, can we do something like this:

`fprintf(stderr, "%s:%d assert failed\n", __FILE__, __LINE__);`

?

Reply:
Done. @apolcyn Thanks for the really good catch !! 🙂